src/ui/FieldTable/FieldValue.ts

```diff
@@ -255,6 +275,12 @@ export class FieldValue extends Component {
     });
   }
 
+  private prependTextCaptionToDom(): void {
+    let elem = this.getValueContainer();
+    elem.insertBefore(this.renderTextCaption(), elem.childNodes[0]);
+    elem.style.display = 'inline-block';
```

Comment:
What would be the best practice you recommend there? Make a global css class for 
`.coveo-display-inline-block { display: inline-block; }`

Or make a css specific to the FieldValue sass?

```
.CoveoFieldValue {
  .coveo-display-inline-block {
    display: inline-block;
  }
}
```